.pre-commit-config.yaml

```diff
@@ -3,6 +3,13 @@ minimum_pre_commit_version: "2"
 repos:
   - repo: local
     hooks:
+      - id: docker
+        name: docker
+        entry: make build-local
+        files: "^fidesctl/"
+        types_or: [file, python]
+        language: system
+
```

Comment:
This diff is unexpected to me. Weren't these changes made on another PR?

Reply:
i merged main and then added this docker build step